stan/math/prim/mat/fun/LDLT_factor.hpp

```diff
@@ -119,10 +119,10 @@ namespace stan {
         return _ldltP->matrixLDLT();
       }
 
-      inline size_t rows() const { return N_; }
-      inline size_t cols() const { return N_; }
+      inline Eigen::DenseIndex rows() const { return N_; }
+      inline Eigen::DenseIndex cols() const { return N_; }
 
-      typedef size_t size_type;
+      typedef Eigen::DenseIndex size_type;
       typedef double value_type;
 
       size_t N_;
```

Comment:
No problem. (I would have fixed it myself if it was easier to create a pull
request to your fork, but that's not so straightforward.)

On Mon, Nov 2, 2015 at 10:35 AM, Joshua Pritikin <notifications@github.com>
wrote:

> In stan/math/prim/mat/fun/LDLT_factor.hpp
> <https://github.com/stan-dev/math/pull/190#discussion_r43640432>:
>
> >
> > -      typedef size_t size_type;
> > +      typedef Eigen::DenseIndex size_type;
> >        typedef double value_type;
> >
> >        size_t N_;
>
> Oh! Sorry for the delay. I realize what Bob had pointed out.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/stan-dev/math/pull/190/files#r43640432>.
>